src/vs/workbench/parts/stats/node/workspaceStats.ts

```diff
@@ -56,11 +56,15 @@ const ModulesToLookFor = [
 	'react',
 	'react-native',
 	'@angular/core',
+	'@ionic',
 	'vue',
+	'tns-core-modules',
 	// Other interesting packages
 	'aws-sdk',
+	'aws-amplify',
```

Comment:
If I remove it, it doesn't get tagged properly (or at least it didn't in my case). It's only getting tagged because it's in `ModulesToLookFor` in lines 473-489.

Reply:
I removed the logic in that commit because lines 486-489 are catching it automatically if it's in `ModulesToLookFor`.

Reply:
As I read these changes, then you removed the `aws` tracking in https://github.com/Microsoft/vscode/pull/59905/commits/0e4ca570a92be0e781831898fc74ca8bdbfa94ba, which means that these modules in `ModulesToLookFor` and the GPDR annotations are leftovers, which also needs to get cleaned up.